py/kubeflow/kfctl/testing/pytests/kf_is_ready_test.py

```diff
@@ -62,16 +61,29 @@ def test_katib_is_ready(record_xml_attribute, namespace):
 
   util.load_kube_config()
 
+  for deployment_name in deployments:
+    logging.info("Verifying that deployment %s started...", deployment_name)
+    util.wait_for_deployment(api_client, namespace, deployment_name, 10)
+
+def test_katib_is_ready(record_xml_attribute, namespace):
   deployment_names = [
     "katib-controller",
     "katib-mysql",
     "katib-db-manager",
     "katib-ui",
   ]
-  for deployment_name in deployment_names:
-    logging.info("Verifying that deployment %s started...", deployment_name)
-    util.wait_for_deployment(api_client, namespace, deployment_name, 10)
+  check_deployments_ready(record_xml_attribute, namespace,
+                          "test_katib_is_ready", deployment_names)
 
+def test_metadata_is_ready(record_xml_attribute, namespace):
```

Comment:
@zhenghuiwang while your here; would you mind replacing test_kf_is_ready with several tests for the remaining things in kf_is_ready?

So maybe

* 1 for central dashboard
* 1 for profiles
* 1 for all the pipelienes stuff (including Argo)
* 1 for TFJob
* 1 For PyTorch
* 1 For Jupyter apps?

Reply:
Done.